iphone/Maps/Common/Common.swift

```diff
@@ -27,18 +28,42 @@ func statusBarHeight() -> CGFloat {
   return min(statusBarSize.height, statusBarSize.width)
 }
 
+private let enableLoggingInRelease = true
```

Comment:
static?

Reply:
In swift variables outside the types are already static. Compiler will warn if I add the `satatic`.